configuration.rst

```diff
@@ -6,92 +6,73 @@ Configuring Symfony (and Environments)
 
 Every Symfony application consists of a collection of bundles that add useful tools
 (:doc:`services </service_container>`) to your project. Each bundle can be customized
-via configuration files that live - by default - in the ``app/config`` directory.
+via configuration files that live - by default - in the ``config/`` directory.
 
-Configuration: config.yml
--------------------------
+Configuration: config/packages/
+-------------------------------
 
-The main configuration file is called ``config.yml``:
+The configuration for a package can be found in ``config/packages``. For
```

Comment:
`can be found in config/packages` --> `can be found in config/packages/`  please always add a trailing slash when referencing directories so the reader never wonders if it's a dir or a file without extension. Thanks!